### PR TITLE
Test improvements

### PR DESCRIPTION
**Pull request by dogweather**
[Trello Card](https://trello.com/c/LvZABXoZ/427-troubleshoot-circleci-and-local-enviroinment-timeout-issues)

Getting Circle CI and local tests stable. It makes the Circle CI tests stable by excluding `js` and `elasticsearch`. In my next PR, I'll work on adding those back in.

### CODE REVIEW

File: bin/spring [hunk 1]
```diff
@@ -4,12 +4,12 @@
 # It gets overwritten when you run the `spring binstub` command.
 
 unless defined?(Spring)
-  require "rubygems"
-  require "bundler"
+  require 'rubygems'
```
Comment (rememberlenny) [inline bin/spring]: @dogweather JSYK, we are using double quotes for everything. Can you update your linter?

**Comment by rememberlenny** [comment]
@dogweather Is this good to go in, without breaking current tests?

File: bin/spring (hunk 1 above)
Comment (dogweather) [inline bin/spring]: Totally; will do.

Comment (dogweather) [inline bin/spring]: Huh, actually, those are auto-generated files, which is why my rubocop settings weren't at issue.

**Comment by dogweather** [comment]
Yep, absolutely, it's good to be merged.

**Pull request merged**